source/sessions/snapshot-sessions.rst

```diff
@@ -233,6 +234,12 @@ Lists of commands that support snapshot reads:
 2. aggregate
 3. distinct
 
+Sending readConcern to the server on all commands
+=================================================
+
+Drivers MUST set the readConcern ``level`` and ``atClusterTime`` fields (as
+outlined above) on all commands in a snapshot session.
```

Comment:
I'd add, "even commands like insert and update that do not accept a readConcern. This ensures the server will return an error for invalid operations, such as writes, within a session configured for snapshot reads." Make the implicit explicit.

Reply:
Done.